config/config.js

```diff
@@ -0,0 +1,27 @@
+require('dotenv').config();
```

Comment:
Why do you have a config.js and config json that are nearly identical.

Reply:
I switched over to a config.js to allow for an env password, but forgot to delete the config.json. Thanks for catching it!